Local install failure

Hello,

I came to this problem while playing with [dva-boilerplate-isomorphic](https://github.com/sorrycc/dva-boilerplate-isomorphic) sample.
Although, the global installation (`npm install -g webstorm-disable-index`) works fine, the local installation fails completely: not even `webstorm-disable-index` directory gets created.

```
[ilema@visva dva-boilerplate-isomorphic]$ npm install

> webstorm-disable-index@1.1.1 postinstall /files/dev/react/dva/dva-boilerplate-isomorphic/node_modules/webstorm-disable-index
> node bin/webstorm-disable-index.js

[webstorm-disable-index] /files/dev/react/dva/dva-boilerplate-isomorphic/node_modules/webstorm-disable-index
[webstorm-disable-index] write files at /files/dev/react/dva/dva-boilerplate-isomorphic/
TypeError: Cannot read property 'split' of undefined
    at Command.run (/files/dev/react/dva/dva-boilerplate-isomorphic/node_modules/webstorm-disable-index/lib/command.js:13:44)
    at next (native)
    at onFulfilled (/files/dev/react/dva/dva-boilerplate-isomorphic/node_modules/co/index.js:65:19)
    at /files/dev/react/dva/dva-boilerplate-isomorphic/node_modules/co/index.js:54:5
    at co (/files/dev/react/dva/dva-boilerplate-isomorphic/node_modules/co/index.js:50:10)
    at toPromise (/files/dev/react/dva/dva-boilerplate-isomorphic/node_modules/co/index.js:118:63)
    at next (/files/dev/react/dva/dva-boilerplate-isomorphic/node_modules/co/index.js:99:29)
    at onFulfilled (/files/dev/react/dva/dva-boilerplate-isomorphic/node_modules/co/index.js:69:7)
    at /files/dev/react/dva/dva-boilerplate-isomorphic/node_modules/co/index.js:54:5
    at co (/files/dev/react/dva/dva-boilerplate-isomorphic/node_modules/co/index.js:50:10)
npm ERR! Darwin 16.3.0
npm ERR! argv "/Users/ilema/.nvm/versions/node/v6.9.1/bin/node" "/Users/ilema/.nvm/versions/node/v6.9.1/bin/npm" "install"
npm ERR! node v6.9.1
npm ERR! npm  v3.10.8
npm ERR! code ELIFECYCLE

npm ERR! webstorm-disable-index@1.1.1 postinstall: `node bin/webstorm-disable-index.js`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the webstorm-disable-index@1.1.1 postinstall script 'node bin/webstorm-disable-index.js'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the webstorm-disable-index package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node bin/webstorm-disable-index.js
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs webstorm-disable-index
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls webstorm-disable-index
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /files/dev/react/dva/dva-boilerplate-isomorphic/npm-debug.log
[ilema@visva dva-boilerplate-isomorphic]$
```

Any ideas, please?

Thank you!

--- a/lib/command.js
+++ b/lib/command.js
@@ -10,7 +10,7 @@ module.exports = class Command {
   * run(cwd) {
     this.cwd = cwd;
     this.pkgInfo = this.getPkgInfo();
-    this.appName = _.last(this.pkgInfo.name.split('/'));
+    this.appName = path.basename(this.cwd);
     this.config = this.pkgInfo.config && this.pkgInfo.config.webstorm || {};
     this.config.index = this.config.index || [];
 
@@ -133,7 +133,7 @@ module.exports = class Command {
     try {
       return require(path.join(this.cwd, 'package.json'));
     } catch (err) {
-      console.warn('[webstorm-disable-index] package.json not found');
+      console.warn('[webstorm-disable-index] package.json not found at ' + this.cwd);
       return { name: 'webstorm-disable-index' };
     }
   }

--- a/test/fixtures/scope/package.json
+++ b/test/fixtures/scope/package.json
@@ -1,5 +1,5 @@
 {
-  "name": "@ali/scope",
+  "name": "@ali/anthoer-name",
   "version": "1.0.0",
   "config": {
     "webstorm": {

--- a/test/lib/command.test.js
+++ b/test/lib/command.test.js
@@ -39,7 +39,7 @@ describe('test/lib/command.test.js', () => {
     command.expect('pkg-config.iml', '<excludeFolder url="file://$MODULE_DIR$/node_modules"/>');
   });
 
-  it('should support scope', function* () {
+  it('should use base name', function* () {
     const command = getCommand();
     yield command.run(path.join(__dirname, '../fixtures/scope'));
     command.expect('modules.xml', '<module filepath="$PROJECT_DIR$/.idea/scope.iml" fileurl="file://$PROJECT_DIR$/.idea/scope.iml"/>');

